src/util/DataModel.js

```diff
@@ -18,17 +18,19 @@ class DataModel {
         this.akaRecord = null;
         this.sendLog = sendLog;
         this.Auth = Auth;
+        this.newGroups = null;
     }
 
     async matchToKartoffel() {
         if (this.isMatchToKartoffel) {
-            this.person_ready_for_kartoffel = await matchToKartoffel(
+            [this.person_ready_for_kartoffel, this.newGroups] = await matchToKartoffel(
```

Comment:
consider the use of object destruction instead of an array, if possible it's preferable